generated/nidcpower/session.py

```diff
@@ -4151,6 +4151,599 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def create_advanced_sequence(self, sequence_name, sequence, set_as_active_sequence=True):
+        '''create_advanced_sequence
+
+        Test.
+
+        Note: Test.
+
+        Args:
+            sequence_name (str): Name of sequence.
+
+            sequence (list of int): Sequence.
```

Comment:
So it looks like a `sequence` is a `Iterable[Dict[str, Any]]` with the `str` being the property name.

Why are we doing a stringly-typed API rather than using enums?  Would we want the `.value` to be the string or the id?

Or at worse, providing constants to the user (and having this function's source use the constants as well)

Reply:
Documentation is incomplete. There will need to be some changes to get the type right. I don't want to put in the effort to fix it in case we end up doing something different.

Reply:
Sorry if I wasn't clear.  This was a "good enough" place for me to open an issue regarding the data format.

Reply:
> Why are we doing a stringly-typed API rather than using enums?
> Would we want the .value to be the string or the id?
> Or at worse, providing constants to the user (and having this function's
> source use the constants as well)
 

We debated the API in #504 

If we had an enum, the client using the Advanced Sequences API would need to refer to attributes differently than how they normally do it:
* `my_session.output_function` when using the propoerty normally.
* `Properties.OUTPUT_FUNCTION` when referring to the same thing in the context of `create_advanced_sequence`.

We thought it would be nicer for the client to be able to simply use `"output_function"` in the context of `create_advanced_sequence`.